data/payment_providers.json

```diff
@@ -468,7 +468,8 @@
     "code": "4bill",
     "vendor": "4bill",
     "payment_method": [
-      "bank_card"
+      "bank_card",
```

Comment:
bank_card => payment_card 

+ payout methods fix.

+ check other methods  for 4bill